backend/features/bootstrap/FeatureContext.php

```diff
@@ -0,0 +1,196 @@
+<?php
+/**
+ * Created by PhpStorm.
+ * User: TEC-GATE
+ * Date: 7/21/2019
+ * Time: 12:59 PM
+ */
+
+use Behat\Behat\Context\Context;
+use Symfony\Component\HttpFoundation\Response;
+
+class FeatureContext implements Context
+{
+    private $api_url = "localhost:8000";
+
+    private $create_endpoint = "/createPainting";
+    private $search_endpoint = "/search";
+    private $latest_endpoint = "/latest/painting";
+    private $delete_endpoint = "/deletePainting";
+    private $update_endpoint = "/updatePainting";
+
+    private $response_code = null;
+    private $response_body = null;
+
+    private $search_response = null;
+
+    // region Painting Create
+
+    /**
+     * @Given /^the backend server is alive$/
+     * @throws Exception
+     */
+
+    public function theBackendServerIsAlive()
+    {
+        $client = new GuzzleHttp\Client(['base_uri' => $this->api_url]);
```

Comment:
let us abstract the usage of GuzzleHttp\Client
by creating our Client , inside it we can use gazzle